concept/type_/plays.rs

```diff
@@ -25,5 +34,36 @@ impl<'a> Plays<'a> {
     pub fn role(&self) -> RoleType<'a> {
         self.role.clone()
     }
+
+    pub fn get_override<'this, Snapshot: ReadableSnapshot>(
+        &'this self,
+        snapshot: &Snapshot,
+        type_manager: &'this TypeManager<Snapshot>,
+    ) -> Result<MaybeOwns<'this, Option<Plays<'static>>>, ConceptReadError> {
+        type_manager.get_plays_overridden(snapshot, self.clone().into_owned())
+    }
+
+    pub fn set_override<Snapshot: WritableSnapshot>(
+        &self,
+        snapshot: &mut Snapshot,
+        type_manager: &TypeManager<Snapshot>,
+        overridden: Plays<'static>,
+    ) {
+        // TODO: Validation
+        type_manager.storage_set_plays_overridden(snapshot, self.clone().into_owned(), overridden)
+    }
+
+    fn into_owned(self) -> Plays<'static> {
+        Plays { player: ObjectType::new(self.player.vertex().into_owned()), role: self.role.into_owned() }
+    }
 }
 
+impl<'a> IntoCanonicalTypeEdge<'a> for Plays<'a> {
+    fn as_type_edge(&self) -> TypeEdge<'static> {
+        build_edge_plays(self.player.vertex().clone().into_owned(), self.role.vertex().clone().into_owned())
+    }
+
+    fn into_type_edge(self) -> TypeEdge<'static> {
+        build_edge_plays(self.player.vertex().clone().into_owned(), self.role.vertex().clone().into_owned())
+    }
+}
```

Comment:
I remember there's  a place in TypeReader / TypeManager where I could use these methods instead of calling the encoding method directly.

Reply:
?